packages/fxa-settings/src/pages/ResetPasswordWithRecoveryKeyVerified/index.tsx

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Localized } from '@fluent/react';
+import { logViewEvent } from '../../lib/metrics';
+import { useNavigate } from '@reach/router';
+import Ready from '../../components/Ready';
+
+type ResetPasswordWithRecoveryKeyVerifiedProps = {
+  serviceName?: string;
+};
+
+const ResetPasswordWithRecoveryKeyVerified = ({
+  serviceName,
+}: ResetPasswordWithRecoveryKeyVerifiedProps & RouteComponentProps) => {
+  const navigate = useNavigate();
+  const viewName = 'reset-password-with-recovery-key-verified';
+
+  return (
+    <>
+      <Ready viewName={viewName} serviceName={serviceName} />
+      <div className="flex justify-center mx-auto m-6">
+        <Localized id="reset-password-with-recovery-key-verified-generate-new-key">
+          <button
+            className="cta-primary cta-xl"
+            onClick={() => {
+              const eventName = `${viewName}.generate-new-key`;
+              logViewEvent(viewName, eventName, {
+                entrypoint_variation: 'react',
+              });
+              navigate('/settings/account_recovery');
+            }}
+          >
+            Generate a new account recovery key
+          </button>
+        </Localized>
+      </div>
+      <div className="links">
+        <Localized id="reset-password-with-recovery-key-verified-continue-to-account">
```

Comment:
We should be using `FtlMsg` instead of `Localized` (I added a note about this in the Storybook acceptance criteria, I'd forgotten to call it out since Dan was finishing up the `l10n.getString()` wrapper). We really need docs about this which we have a ticket filed for but we haven't gotten to of course 🤦‍♀️